### PR TITLE
Trello Integration

### PR DESCRIPTION
**Pull request by LukasRos**
Implementation of Trello script as used for the integration guide; based on the existing Jira script.

### CODE REVIEW

**Comment by sukottokun** [comment]
@greg-1-anderson Can you look this over? I can add you to a site that has this working if you like.

**Comment by rachelwhitton** [comment]
+1

**Comment by greg-1-anderson** [comment]
I'm out of town for about a week; I can look at this and #132 when I get back.

File: trello_integration/trello_integration.php [hunk 1]
```diff
@@ -0,0 +1,145 @@
+<?php
+
+// Get the environment; we will post a new comment to Trello each time
+// a commit appears on a new branch on Pantheon.
+$env = $_ENV['PANTHEON_ENVIRONMENT'];
+
+// Do not watch test or live, though.
+if (($env == 'live') || ($env == 'test')) {
```
Comment (greg-1-anderson) [inline trello_integration/trello_integration.php]: In the case of Trello, these commits never set any sort of status on the Trello card, right? In this instance, then, it makes sense to track activity on the card in every multidev that the commit moves through, as the current implementation does.

**Comment by greg-1-anderson** [comment]
+1 from me, pending @rachelwhitton's satisfaction with the resolution of my comments.

File: trello_integration/trello_integration.php (hunk 1 above)
Comment (LukasRos) [inline trello_integration/trello_integration.php]: Removed those lines so the script triggers in every environment.

**Pull request merged**